examples/azure-rules-perf-optimized.yaml

```diff
@@ -20,6 +20,23 @@ groups:
           Install a SUSE supported OS release
         scored: true
       - id: 2.1.2
+        description: "Check os release"
+        audit: 'echo VERSION_ID=$(cat /etc/os-release | grep VERSION_ID= | sed "s/[^0-9]//g")'
+        tests:
+          test_items:
+            - flag: 'VERSION_ID'
+              compare:
+                op: gte
+                value: 151
```

Comment:
@aleksei-burlakov this is on the right track for the version number, but we need to add something that verifies that this is SLES4SAP and not another distribution!

Reply:
This check was added in #30 where we check the product. This PR is only aimed at the version.

Reply:
Ohh, okay!